scripts/make-react.js

```diff
@@ -5,9 +5,15 @@ const fs = require("fs");
 const { program } = require("commander");
 const path = require("path");
 const prettier = require("prettier");
-const pascalCase = require("pascal-case");
```

Comment:
please remove pascal case from build script as well, its giving errros and warnings.

Reply:
can you specify, what errors and warnings? If its not tenable maybe I can write a simple algorithm that fits our specific use case.

Reply:
my bad!